src/pages/BDI_DataImport.page

```diff
@@ -83,7 +104,20 @@
                                 <div class="slds-form slds-form_stacked slds-m-left_large">
                                     <div class="slds-form-element">
                                         <div class="slds-form-element__control">
-                                            <apex:commandButton value="{!$Label.bdiRunBtn}" action="{!importData}" immediate="false" styleClass="slds-button slds-button_brand" rerender="idForm"/>
+                                            <apex:actionFunction name="importDataActionFunction"
+                                                                 action="{!importData}"
+                                                                 oncomplete="buttonsEnabled(true);"
+                                                                 rerender="idForm">
+                                            </apex:actionFunction>
+
+                                            <apex:commandButton value="{!$Label.bdiRunBtn}"
+                                                                action="{!importData}"
+                                                                immediate="false"
+                                                                styleClass="slds-button slds-button_brand"
+                                                                rerender="idForm"
+                                                                id="idDataImportProcessButton"
+                                                                onclick="return tryImportData();"/>
```

Comment:
I've been thinking about this a bit more - I think you may be able to simplify this overall by specifying:
  onclick="buttonsEnabled(false);"
  onComplete="buttonsEnabled(true);"/>

in the attributes of the command button, and I think you can avoid the actionFunction entirely, as well as the tryImportData() javascript function. That seems a bit more robust as it calls the apex controller in the more traditional visualforce method instead of from the javascript intermediary - however, my visualforce is pretty rusty so feel free to let me know if you are ahead of me and have justification for this setup that I wasn't are of yet.